src/NumericalAlgorithms/Interpolation/InterpolationTarget.hpp

```diff
@@ -94,6 +96,19 @@ namespace intrp {
 ///      Domain will be filled with this value. If this variable is not defined,
 ///      then the `apply` function must check for invalid points,
 ///      and should typically exit with an error message if it finds any.
+/// - should_interpolate (optional):
+///      A template function that takes as a single argument a const reference
+///      to the \ref DataBoxGroup, and returns a `bool`: `true` if the
+///      element should interpolate in the current state, `false` otherwise.
+///      Often, this will examine the `temporal_id` to determine whether to
+///      interpolate, but it may examine any part of the \ref DataBoxGroup.
+///      If omitted, the behavior is as if the `should_interpolate` function
+///      always returns true. This function is called using the DG element's
+///      \ref DataBoxGroup and is only used if `InterpolateToTarget` is
+///      in the element's action list.
+/// - interpolating_dg_component (required if not `is_sequential`):
+///      A type alias for the DG element array that will be interpolating to the
```

Comment:
Just an observation: if `is_sequential`, then the thing that interpolates to the interpolation target might not be a DG element array.  Not sure if I want a change here, but the current wording might imply that the thing interpolating is always a DG element array, which might be confusing.